packages/runtime/container-runtime/src/blobManager/blobManager.ts

```diff
@@ -475,6 +501,27 @@ export class BlobManager {
 		});
 	}
 
+	private createBlobPlaceholder(blob: ArrayBufferLike): IFluidHandleInternal<ArrayBufferLike> {
+		const localId = this.localBlobIdGenerator();
+
+		return new BlobHandle(
```

Comment:
how does this work while attaching?

Reply:
Not sure I follow the question - while attaching what specifically?

Reply:
look at createBlobLegacy, it waits to upload while attaching

Reply:
Sure, we can do the same in the new flow.

Reply:
Longer-term maybe we should look into why this was needed, since it'll block making the API sync if we ever want to do that.

Reply:
#24184